server/src/main/scala/controllers/Application.scala

```diff
@@ -242,14 +242,11 @@ class Application @Inject()(implicit val config: Configuration, env: Environment
           )(new Timeout(30 seconds))
 
         portStateFuture.map {
-          case Some(portState: PortState) =>
+          case portState: PortState =>
```

Comment:
does this need a recover on it now?

Reply:
yep, nice catch (it should have had one before too)